components/store/OnlineStoreComp.tsx

```diff
@@ -0,0 +1,314 @@
+                            mt: "0.3rem",
+                          }}
+                        ></Box>
+                      </CardContent>
+                      {address.zipCode != null ? (
+                        <Grid
+                          sx={{
+                            display: "flex",
+                            alignSelf: "end",
+                            height: "full",
+                            width: "auto",
+                            // bgcolor: "red",
+                            flexDirection: "column",
+                          }}
+                        >
+                          <Box
+                            sx={{
+                              ...all_center,
+                              m: "0.7rem",
+                              height: { xl: "42px" },
+                              width: { xl: "45px" },
+                              borderRadius: "10px",
+                              bgcolor: theme.palette.primary.main,
+                              color: theme.palette.common.white,
+                              fontSize: theme.typography.subtitle1.xl,
+                            }}
+                          >
+                            <SendIcon />
+                          </Box>
+                        </Grid>
+                      ) : 
+                      
+                      (
+                        <></>
+                      )}
+                    </Card>
+                  </Grid>
+                );
+              }
+            )}
+    
+              {currentItemsLength == currentResponseTotalCount ? (<></>) : (<><Box
+              component={"div"}
+              sx={{ ...all_center, height: "7rem", width: "100%" }}
+            >
+              <Button
+              onClick={()=>dispatch(storePageNumber({pageNumber : pageNumber + 1}))}
+                sx={{
+                  height: "2.5rem",
+                  width: "8rem",
+                  alignSelf: "center",
+                  background: theme.gradient_color.button_color,
+                  //   mt: "1rem",
+                  color: theme.palette.common.white,
+                  fontSize: theme.typography.subtitle2.xl,
+                  "&:hover": {
+                    background: theme.gradient_color.button_hover_color,
+                    color: theme.palette.common.black,
+                  },
+                }}
+              >
+                <Typography>Load More</Typography>
+              </Button>
+            </Box></>)}
+        
+            
+            </>) 
+            
+            : 
+            
+            (<>
+            <Grid xl={12} sx={{...all_center, height:{xl:"10rem"}}}>
+              <Typography sx={{fontSize:theme.typography.h5.lg, fontWeight:'500'}}>
+                No store available
+              </Typography>
+            </Grid>
+            </>)
+
+
+        )}
+
+
+
+
+      </Box>
+    </Grid>
+  );
+};
+
+export default OnlineStoreComp;
```

Comment:
Use Full name ` OnlineStoreComp`  Component or you don't need to add every where `Component` suffix in every component